package.json

```diff
@@ -1,18 +1,34 @@
 {
   "private": true,
   "name": "care-ops-frontend",
-  "version": "0.1.0",
+  "version": "1.0.0",
   "description": "Frontend Assets for RoundingWell Care Team Operations",
   "repository": "git@github.com:RoundingWell/care-ops-frontend.git",
   "author": "RoundingWell",
   "license": "CC-BY-4.0",
   "homepage": "https://RoundingWell.com",
   "type": "module",
   "scripts": {
+    "analyze": "npx vite-bundle-visualizer",
+    "prebuild": "npm i",
+    "build": "vite build",
+    "precoverage": "vite build -m test",
+    "coverage": "npm run coverage:clear & vite preview -m test & cypress run --env COVERAGE=true && npm run coverage:report",
+    "precoverage:component": "vite build -m test",
+    "coverage:component": "npm run coverage:clear & vite preview -m test & cypress run --component --env COVERAGE=true && npm run coverage:report",
     "coverage:clear": "npx rimraf coverage",
     "coverage:report": "nyc report --reporter=html --reporter=text",
     "coverage:report:ci": "nyc report --reporter=lcovonly || true",
+    "dev": "vite -d --force",
+    "test": "npm run coverage:clear & vite --force -m test & cypress open --env COVERAGE=true",
```

Comment:
@nmajor25 the biggest change to day-to-day might be the script naming.

Essentially `npm run dev:coverage` -> `npm test`
`npm start` will run a prod build, but it will not update on code changes.  It's notable with Vite that the dev build and the prod build are very different.  However we'll run our CI tests off of a prod build as well as the `npm run coverage` tests